CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,20 @@
+# openFrameworks Code of Conduct
+
+One of the goals of openFrameworks is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, socioeconomic status, age, level of experience, personal appearance, ethnicity, and religion (or lack thereof).
+
+As the openFrameworks community, we pledge to respect all members who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, engage in discussion on the forum, mailing list or IRC, participate in meetups and workshops, or who are involved in any way with openFrameworks.
+
+Examples of unacceptable behavior includes the use of sexual language or imagery, derogatory comments or personal attacks, trolling, public or private harassment, insults, or other disrespectful conduct. We promise each other to:
+
+* Participate in an authentic and active way. In doing so, we contribute to the health and longevity of this community.
+* Exercise consideration and respect in our speech and actions.
+* Attempt collaboration before conflict.
+* Be mindful of other community members.
+
+Alert anyone in [the core](https://github.com/orgs/openframeworks/teams/core) if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
```

Comment:
[the core](https://github.com/orgs/openframeworks/teams/core) This link points to 404.